src/services/kinesis.ts

```diff
@@ -44,6 +50,6 @@ export async function getLedgerStream(connection: Connection, cursor = 'now'): P
 
 export async function getAccount(connection: Connection, accountId: string) {
   const server = getServer(connection)
-  const account: AccountRecord = await server.accounts().accountId(accountId).call() as any
+  const account: AccountRecord = await server.loadAccount(accountId)
```

Comment:
This returns slightly different things. `loadAccount()` returns an `AccountResponse` variable.

Reply:
`AccountResponse` basically is an `AccountRecord` with some more fields though.